ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml

```diff
@@ -1,9 +1,26 @@
 pragma Singleton
 
 import QtQuick 2.13
+import utils 1.0
 
 QtObject {
+    id: root
+    property var profileSectionModuleInst: profileSectionModule
+    property var profileModule:  profileSectionModuleInst.profileModule
     property var onBoardingModul: onboardingModule
```

Comment:
Maybe we can address that typo here where all the onboarding new flow is being building and integrating. See https://github.com/status-im/status-desktop/pull/5112#discussion_r833304502

Reply:
this was introduced in 30f11c06d4, but yeah, changed